OutOfSchool/OutOfSchool.WebApi/Startup.cs

```diff
@@ -211,8 +211,11 @@ public void ConfigureServices(IServiceCollection services)
             var swaggerConfig = Configuration.GetSection(SwaggerConfig.Name).Get<SwaggerConfig>();
 
             // Add feature management
-            services.AddFeatureManagement(Configuration.GetSection(Constants.SectionName));
-            services.AddSingleton<IConfiguration>(Configuration);
+            services.AddFeatureManagement(Configuration.GetSection(FeatureManagementConfig.Name));
+            services.AddSingleton<FeatureManagementConfig>((serviceProvider) =>
+            {
+                return Configuration.GetSection(FeatureManagementConfig.Name).Get<FeatureManagementConfig>();
```

Comment:
I guess here you can use services.Configure<FeatureManagementConfig>(Configuration.GetSection(FeatureManagementConfig.Name));
And then get it from your controller as 
public FeatureManagementController(IOptions<FeatureManagementConfig> featureManagementConfig)
{
 this.featureManagementConfig = featureManagementConfig.Value;
}